robotoff/prediction/ocr/core.py

```diff
@@ -12,6 +12,7 @@
 from robotoff.utils.types import JSONType
 
 from .brand import find_brands
+from .category_from_AOC import find_category_from_AOC
```

Comment:
I think we shall call the module category and the function find_categories, for we only have one function by prediction type. Maybe in the future we want to detect using not only AOC.